packages/gestalt-design-tokens/tokens/vr-theme/base/rounding.json

```diff
@@ -1,49 +1,9 @@
 {
   "base": {
     "rounding": {
-      "0": {
-        "type": "borderRadius",
-        "value": "0px"
-      },
-      "4": {
-        "type": "borderRadius",
-        "value": "4px"
-      },
-      "8": {
-        "type": "borderRadius",
-        "value": "8px"
-      },
-      "12": {
-        "type": "borderRadius",
-        "value": "12px"
-      },
-      "16": {
-        "type": "borderRadius",
-        "value": "16px"
-      },
-      "20": {
-        "type": "borderRadius",
-        "value": "20px"
-      },
-      "24": {
-        "type": "borderRadius",
-        "value": "24px"
-      },
-      "28": {
-        "type": "borderRadius",
-        "value": "28px"
-      },
-      "32": {
-        "type": "borderRadius",
-        "value": "32px"
-      },
-      "999": {
-        "type": "borderRadius",
-        "value": "999px"
-      },
-      "circle": {
-        "type": "borderRadius",
-        "value": "50%"
+      "unit": {
```

Comment:
whats this?

Reply:
It's the base unit (in theory the tokens would be using math) (e..g sema.100 = base.unit * 1 ). 

It's supported by token studio, but not in our build system yet